paasta_tools/kubernetes_tools.py

```diff
@@ -479,6 +479,8 @@ def format_kubernetes_app(self) -> V1Deployment:
 
         code_sha = get_code_sha_from_dockerurl(docker_url)
         complete_config = V1Deployment(
+            api_version='apps/v1',
+            kind='Deployment',
```

Comment:
Can you please double check that we need to set the `kind` as well?
I expect k8s to able to deduce the kind from `V1Deployment`.

Reply:
Actually, looking at `V1` part in the `V1Deployment`, shouldn't k8s deduce the version as well?